src/daemon/rpc_command_executor.cpp

```diff
@@ -92,6 +92,19 @@ namespace {
     return s + " " + (t > now ? "in the future" : "ago");
   }
 
+  std::string get_time_hms(time_t t)
+  {
+    int hours, minutes, seconds;
+    char buffer[24];
+    hours = t / 3600;
+    t %= 3600;
+    minutes = t / 60;
+    t %= 60;
+    seconds = t;
+    sprintf(buffer, "%02d:%02d:%02d", hours, minutes, seconds);
```

Comment:
I'd rather snprintf to guard against future code changes. %02d would print lots of chars for a large positive or negative number. %u and unsigned ints would also fit better.